src/libraries/Microsoft.VisualBasic.Core/Directory.Build.props

```diff
@@ -2,6 +2,7 @@
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
     <AssemblyVersion>$([MSBuild]::Add($(MajorVersion), 5)).$(MinorVersion).0.0</AssemblyVersion>
+    <MajorVersion>$([MSBuild]::Add($(MajorVersion), 6))</MajorVersion>
```

Comment:
NIT: Should we add a comment or something here to a) explain why we are changing these two so it doesn't get removed and b) say that the order of these two properties is important since if you switch them now MajorVersion used on assembly version will be wrong?

Reply:
Sure I'll add a comment.

Reply:
I think this should only be +5.  AssemblyVersion and FileVersion should match if possible.  We need to make sure the following:
Once this change is ported to 6.0 both AssemblyVersion and FileVersion are higher than 5.0, AssemblyVersion doesn't change from GA, and ideally Major versions of Assembly and File match.
7.0 has higher major version that 6.0.

Also, can we get away with just changing MajorVersion and then let other targets construct the correct Assembly and File versions?

Reply:
Ok, so for reference here are the versions we previously shipped

Release | File Version | Assembly Version
-------- | ------------- | -------------------
3.1 | 4.7.x.y | 10.0.5.0
5.0 | 11.0.x.y | 10.0.6.0
6.0 | 6.0.x.y | 11.0.0.0
7.0 | 7.0.x.y | 12.0.0.0

What about making 7.0 simpler, and just special casing 6.0 servicing?

Reply:
> Also, can we get away with just changing MajorVersion and then let other targets construct the correct Assembly and File versions?

Unfortunately we can't as we set `AssemblyVersion` on `Versions.props` for all projects to `Major.Minor.0.0` and at that point `Major.Minor` have whatever values are set on `Versions.props` in the case of main, 7.0, so we have to override `AssemblyVersion` on the project directly.

Reply:
Done, I added a comment and also made FileVersion Major and Minor match the assembly version, then on 6.0 we can just bump the minor version and that way going forward FileVersion will match the assembly version.